Домашнее задание 5/Домашнее задание 5/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Домашнее_задание_5
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+
+            double a = Program.Formula(2, 5);
+            double b = Program.Formula(3, 7);
+            double c = Program.Formula(5,11);
+
+            double x = a + b + c;
```

Comment:
Зачем запоминать возвращаемое методом значение в переменной, если далее оно используется только один раз? Лучше сразу написать вызов в нужном месте.
Кроме того, если вы вызываете метод класса внутри этого класса, то имя класса перед именем метода можно не писать.
Заметьте, во втором слагаемом три константы разные! То есть выделенный метод должен использовать три параметра
```suggestion
            double x = Formula(2, 5, 5) + Formula(3, 7, 5) + Formula(5,11, 11);
```